website/docs/registry/faq.mdx

```diff
@@ -13,6 +13,10 @@ If you have issues formatting documentation, use the [documentation preview tool
 
 If you have already released a version of your provider and are unhappy with its documentation, [fix the documentation issue in your repository and include it in a new release](#how-do-i-correct-bugs-in-published-versions).
 
+### Why is my documentation not rendered like it is in Github?
+
+We make every effort to support GitHub Markdown syntax in the registry documentation, but there may be instances where compatibility limitations arise. To ensure consistent rendering in the Public Registry, we recommend using syntax widely supported across various Markdown flavors. Use the [documentation preview tool](https://registry.terraform.io/tools/doc-preview) to view how the Terraform Registry will format your documentation.
```

Comment:
We try to only capitalize products in the docs, and another teeny suggestion for conciseness:
```suggestion
We make every effort to support GitHub Markdown syntax in the registry documentation, but occasionally compatibility limitations arise. To ensure consistent rendering in the public registry, we recommend using syntax widely supported across various Markdown flavors. Use the [documentation preview tool](https://registry.terraform.io/tools/doc-preview) to view how the Terraform Registry will format your documentation.
```